static/styles/portico/portico.css

```diff
@@ -702,6 +701,31 @@ input.text-error {
     margin-left: -3px;
 }
 
+.skip-link {
+    background: hsl(153, 32%, 55%);
+    top: -100px;
+    left: 300px;
+    position: absolute;
+    margin-left: 5px;
+    overflow: hidden;
+    border-radius: 6px;
+    padding: 6px 10px;
+    cursor: pointer;
+    color: hsl(0, 0%, 98%);
+    font-size: 18px;
+    font-weight: bold;
+    outline: 5px auto hsl(152, 40%, 42%);
+    &:focus {
+        z-index: 5000;
+        transition: ease-in-out 0.3s;
+        top: -3px;
+        width: auto;
+        height: 25px;
+        outline: 5px auto hsl(152, 40%, 42%);
+        color: hsl(0, 0%, 98%);
+    }
+}
+
 .app {
```

Comment:
Can you talk about how you have tested this CSS (and made sure to avoid duplication) in the commit message?

Reply:
I have read about the documentation of HTML and CSS. But since I have only added one class which is unique. But i have checked which fields are affecting the issue. If there is any can you suggest me ?

Reply:
What I meant was, you should check if there are any existing classes which you can reuse. And defining totally new colors colors should be done thoughtfully, so as to avoid duplication. I don't have any specific suggestions right now.

Reply:
Ok, I'll check for the classes which are similar to the one I need.  In the button I have added some animation due to which I created new class. But for my future issue I'll make sure that there are no duplicate css used.